src/main/resources/anet-schema.yml

```diff
@@ -94,6 +94,15 @@ definitions:
             params:
               type: array
           required: [type]
+      aggregation:
+        type: object
+        properties:
+          widget:
+            type: string
+            enum: [likertScale, numberAggregation, default]
```

Comment:
Only one of these three does aggregation.

Reply:
Will be picked up in #2971